src/components/Dialogs/Dialog/Dialog.tsx

```diff
@@ -21,40 +26,57 @@ export interface DialogProps extends ModalProps {
     acceptButton: ButtonProps;
     /** Properties of the cancel button */
     cancelButton?: ButtonProps;
+    /** when set the content will have a scroll bar if it's longer than the value set */
+    maxHeight?: string | number;
```

Comment:
doesn't have default value? Otherwise anyone can pass any value. I think there are only two possibilities, wrapped and fixed height